src/pages/tests/Login.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Login from '../Login';
+import { ThemeProvider } from '@emotion/react';
+import { theme } from '@/theme';
+import { toast } from 'react-toastify';
+
+vi.mock('react-router-dom', async () => {
+  const actual = await vi.importActual<typeof import('react-router-dom')>('react-router-dom');
+  return {
+    ...actual,
+    useNavigate: () => vi.fn(),
+  };
+});
+
+vi.mock('@/hooks/useUserInfo', () => {
+  return {
+    __esModule: true,
+    default: () => ({
+      isValid: false,
+      setUser: vi.fn(),
+    }),
+  };
+});
+
+vi.mock('react-toastify', () => ({
+  toast: { warn: vi.fn() },
+}));
+
+describe('Login Page', () => {
+  beforeEach(() => {
+    render(
+      <MemoryRouter>
+        <ThemeProvider theme={theme}>
+          <Login />
+        </ThemeProvider>
+      </MemoryRouter>
+    );
+  });
+
+  it('이메일/비밀번호 입력창, 버튼 렌더링 테스트', () => {
+    expect(screen.getByPlaceholderText(/이메일/i)).toBeInTheDocument();
+    expect(screen.getByPlaceholderText(/비밀번호/i)).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /로그인/i })).toBeInTheDocument();
+  });
+
+  it('빈 값에서 blur 시 에러 메시지 출력 테스트', async () => {
+    const emailInput = screen.getByPlaceholderText(/이메일/i);
+    fireEvent.focus(emailInput);
+    fireEvent.blur(emailInput);
```

Comment:
학습을 위해 `fireEvent`와 [userEvent](https://testing-library.com/docs/user-event/intro/)를 각각 어떤 상황에서 사용하면 좋을지 고민해보시면 좋겠어요
```suggestion
    const user = userEvent.setup();

    await user.click(emailInput);
    await user.tab();
```